geoips_plugin_example/__init__.py

```diff
@@ -9,3 +9,8 @@
 # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+"""Init file for geoips_plugin_example."""
+
+from .version import __version__, __version_tuple__
+
+__all__ = ["__version__", "__version_tuple__"]
```

Comment:
@lance-wilson Note I added "__all__" so the code_checks passed (it was failing on unused __version__)